### ISSUE TITLE
Create game option from LAN is greyed out in CS1.6 on Vita

### ISSUE THREAD

**Issue by GooSee2001** [FWGS/xash3d-fwgs#1428]
Downloaded all the required libraries (I already had them on my previous xash3d) and still can’t create a bot server because Create game from LAN Game option is greyed out. It works on Half-Life though and worked on old xash3d port.

**Comment by Maverick0919** [comment]
Yeah same, ever since I switched to the newer version. Have you found a fix yet? I might try replacing the client with one I downloaded for the older version

**Comment by a1batross** [comment]
Can you post engine.log?

**Comment by Maverick0919** [comment]
Hi, how do I do that?

**Comment by a1batross** [comment]
Should work by now. If it doesn't you might probably ask @SNMetamorph for help, since he worked on Vita port lately, including cs16-client port.

**Comment by SNMetamorph** [comment]
No, it won't work as for now because there are porting issues with server library. Not sure is it caused by YaPB or ReGameDLL, but in theory it is possible to use regamelite instead.
